editor/debugger/editor_profiler.cpp

```diff
@@ -510,13 +510,18 @@ void EditorProfiler::_bind_methods() {
 }
 
 void EditorProfiler::set_enabled(bool p_enable, bool p_clear) {
-	activate->set_pressed(false);
 	activate->set_disabled(!p_enable);
 	if (p_clear) {
 		clear();
 	}
 }
 
+void EditorProfiler::set_pressed(bool p_pressed) {
+	activate->set_pressed(p_pressed);
+
```

Comment:
This blank line could be removed.

Reply:
done. for both Profiler and Visual Profiler.